spec/lib/jobs/create-default-pollers-spec.js

```diff
@@ -72,7 +72,7 @@ describe("Job.Pollers.CreateDefault", function () {
             expect(waterline.catalogs.findMostRecent.firstCall.args[0])
                 .to.have.property('node', nodeId);
             expect(waterline.catalogs.findMostRecent.firstCall.args[0])
-                .to.have.property('source', 'bmc');
+                .to.have.property('source').that.is.an('object');
```

Comment:
I think the assertion just for "object" type is not enough, we'd expect both bmc & rmm is checked.